docs/man/rpm-rpmrc.5.scd

```diff
@@ -0,0 +1,100 @@
+RPM-RPMRC(5)
+
+# NAME
+*rpm-rpmrc* - rpm platform compatibility configuration
+
+# SYNOPSIS
+_VARIABLE_: {_ARCH_|_OS_}: _VALUE_ ...
+
+_VARIABLE_: _ARCH_ _VALUE_
+
+# DESCRIPTION
+The low-level machine architecture and OS configuration in *rpm* is
+managed via a set of rpmrc files as defined by the rpmrc path.
+Most users never need to look at, much less touch these files.
+
+Each file in the colon separated rpmrc path is read sequentially by
+*rpm* for configuration information. Tildes will be expanded to the
+value of the environment variable *HOME*.
+The first file in the path must exist, others are considered optional.
+
+If a value is defined multiple times, the last entry wins. The default
+rpmrc path uses this to achieve the following hierarchy of platform
+configuration:
+. Generic *rpm* factory defaults
+. Vendor (distribution) specific configuration
+. Host specific configuration
+. User specific configuration
+
+In older rpm versions the path of per-user rpmrc was _~/.rpmrc_.
+This is still processed if it exists and the new configuration directory
+does not exist.
+
+# CONFIGURATION
+_ARCH_ and _OS_ relate to *uname*(2) machine and operating system
+information, but are not 1:1 equivalent.
+The following directives are supported in the rpmrc files:
+
+*arch_canon* _ARCH_: _CANON_ARCH_ _ARCH_NUM_
```

Comment:
I'm still confused by this directive (so the description perhaps doesn't do justice) :sweat_smile: As in, what is actually being aliased to what. Is it `_ARCH_` being declared as an alias to `_CANON_ARCH_`?

Reply:
Eh, after reading the line that follows once again ("CANON_ARCH is what the ARCH entries in other rpmrc directives refer to."), I think I'm getting it now :smile: ARCH is indeed an alias to CANON_ARCH.